db/migrations/postgres/000090_add_namespace_contractindex.up.sql

```diff
@@ -0,0 +1,3 @@
+BEGIN;
+ALTER TABLE namespaces ADD COLUMN firefly_contracts TEXT;
```

Comment:
Can we rename the filenames for the new richer JSON object we're storing `contractindex`=>`fireflycontracts`